app/config/routing.yml

```diff
@@ -0,0 +1,8 @@
+eduard_trayan_website:
+    resource: "@EduardTrayanWebsiteBundle/Resources/config/routing.yml"
+    prefix:   /
+    
+eduard_trayan_common:
```

Comment:
Please remove this importing because common controller doesn't provide any routes